token/core/fabtoken/issuer.go

```diff
@@ -50,11 +54,14 @@ func (s *Service) Issue(issuerIdentity view.Identity, typ string, values []uint6
 		nil
 }
 
+// VerifyIssue checks if the outputs of an IssueAction match the passed tokenInfos
 func (s *Service) VerifyIssue(tr driver.IssueAction, tokenInfos [][]byte) error {
 	// TODO:
```

Comment:
please address this todo

Reply:
Let's address this TODO once we clarify the function contract at the interface level

https://github.com/hyperledger-labs/fabric-token-sdk/blob/17c5c486cc1c918837c0154744f9cfaf8fb5a89a/token/driver/issue.go